pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -233,6 +235,45 @@ def test_all(self):
                 )
 
 
+class EPELSyncTestCase(unittest.TestCase):
+    """Sync large EPEL repository."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Skip test if the test is running in Travis-CI.
+
+        See: https://docs.travis-ci.com/user/environment-variables/
+
+        TRAVIS environment variable is available to all builds.
+        """
+        cls.travis = 'TRAVIS' in os.environ
+
+    @skip_if(bool, 'travis', True)
+    def test_sync_large_repo(self):
+        """Sync large EPEL repository."""
+        cfg = config.get_config()
+        client = api.Client(cfg, api.json_handler)
+
+        repo = client.post(REPO_PATH, gen_repo())
+        self.addCleanup(client.delete, repo['_href'])
+
+        remote = client.post(
+            RPM_REMOTE_PATH,
+            gen_rpm_remote(url=RPM_EPEL_URL)
+        )
+        self.addCleanup(client.delete, remote['_href'])
+
+        # Sync the repository.
+        self.assertIsNone(repo['_latest_version_href'])
+        sync(cfg, remote, repo)
+        repo = client.get(repo['_href'])
+        added_content = get_added_content(repo)
+        self.assertIsNotNone(
```

Comment:
```assertIsNotNone``` might be the wrong method to use since ```len(added_content(...))``` will never be ```None```.   ```0 is None``` asserts False, so it really should just check if ```len(added_content(...)) > 0```.

Reply:
Let me check this.